components/tabs.tsx

```diff
@@ -0,0 +1,25 @@
+import * as RadixTabs from '@radix-ui/react-tabs'
+import * as React from 'react'
+
+export const Tabs = (props: any) => (
+  <RadixTabs.Root {...props} className={`py-4 ${props.className}`} />
+)
+
+export const TabsHeaders = (props: any) => (
+  <RadixTabs.List
+    {...props}
+    className={`flex space-x-4 border-b-1 border-solid border-gray-200 px-4 ${
+      props.className || ''
+    }`}
+  />
+)
+
+export const TabHeader = (props: any) => {
+  return (
+    <RadixTabs.Trigger {...props} className={`pb-4${props.className || ''}`} />
+  )
+}
+
+export const TabContent = (props: any) => (
+  <RadixTabs.Content {...props} className={`px-4 ${props.className || ''}`} />
+)
```

Comment:
break these up into one component per file, pls. if you want to keep things clean create a `tabs` directory in components with an `index.ts`:
```ts
export * from './tabs'
export * from './tabs-headers'
...
```
and then in `~/components/index.ts`:
```ts
...
export * from './tabs'
```

Reply:
If they were more complicated, I might agree, but these are fairly simple and you're never going to use any of them without using all of them. Unless you feel very strongly about this, I'd prefer to leave them in a single file.